client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -152,6 +152,28 @@ class ShippingLabels extends Component {
 		);
 	};
 
+	renderAddCardExternalInfo = () => {
+		const { masterUserWpcomLogin, masterUserEmail, translate } = this.props;
+
+		if ( ! masterUserWpcomLogin ) {
+			return null;
+		}
+
+		return (
+			<p>
+				{ translate(
+					'Credit cards are retrieved from the following WordPress.com account: %(wpcomLogin)s (%(wpcomEmail)s)',
```

Comment:
Nothing particularly important about this, but I noticed that we're writing what is almost exactly a standard email name/address pair but we're using `(` and `)` instead of angle brackets…

```js
%(wpcomLogin)s <%(wpcomEmail)s>
```

take it for whatever you want it to be 😉

Reply:
it's what [email itself uses](https://tools.ietf.org/html/rfc822) as a `phrase` before a mail spec